test/fixtures/books.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
You can delete this file, we won't need it

Reply:
Done